src/shared_modules/utils/windowsHelper.h

```diff
@@ -141,6 +144,80 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(std::string timestamp1, std::string timestamp2)
```

Comment:
`timestamp1` will never be modified so, it should be `cont`. It is also better to pass the object as reference to avoid creating a new unnecessary object, the only exception is for the fundamental types (char, int, double, etc).

```suggestion
    static std::string normalizeTimestamp(const std::string& timestamp1, std::string timestamp2)
```

Reply:
Changed!